modules/structs/repo.go

```diff
@@ -96,6 +96,7 @@ type Repository struct {
 	AllowRebaseUpdate             bool             `json:"allow_rebase_update"`
 	DefaultDeleteBranchAfterMerge bool             `json:"default_delete_branch_after_merge"`
 	DefaultMergeStyle             string           `json:"default_merge_style"`
+	DefaultAllowMaintainerEdit    bool             `json:"default_allow_edits_from_maintainers"`
```

Comment:
`json:"default_allow_maintainer_edit"` to match the name?

And there is a `make generate-swagger` command to re-generate the API document.

Reply:
Thanks, done.